tests/manager/manager-executeQuery_error-003.phpt

```diff
@@ -24,7 +24,7 @@ try {
 ===DONE===
 <?php exit(0); ?>
 --EXPECT--
-MongoDB\Driver\Exception\CommandException(2): unknown top level operator: $foo
```

Comment:
Why was this change needed? Testing with 5.0 locally, this error message/code appears to still be returned for a query with `{$foo: 1}`.

While we're changing this, I would suggest using `$unsupportedOperator` to make the test more self-documenting.

Reply:
Here's a diff from the test run:
```diff
- BulkWriteException: unknown top level operator: $foo
+ BulkWriteException: unknown top level operator: $foo. If you have a field name that starts with a '$' symbol, consider using $getField or $setField.
```

I'll use `$unsupportedOperator` instead.

Reply:
Oh, interesting. That must have been added in the rc0 release. I was still running an earlier 5.0 version locally.